Aiolos/Aiolos/Sources/Panel.swift

```diff
@@ -94,16 +94,22 @@ public extension Panel {
     override func viewDidLayoutSubviews() {
         super.viewDidLayoutSubviews()
 
-        let (resizeFrame, panelFrame) = self.view.bounds.divided(atDistance: ResizeHandle.Constants.height, from: .minYEdge)
-        self.resizeHandle.frame = resizeFrame
-        self.panelView.frame = panelFrame
-
-        let lineWidth = 1.0 / UIScreen.main.scale
-        var dividerFrame = panelFrame.insetBy(dx: lineWidth, dy: 0.0)
-        dividerFrame.size.height = lineWidth
-        dividerFrame.origin.y -= dividerFrame.size.height / 2.0
-        self.separatorView.frame = dividerFrame
-
+        if configuration.shouldHideHandleSpace {
+            resizeHandle.frame = .zero
```

Comment:
Instead of using `.zero` (which defaults to [(0,0),(0,0)] I would prefer to use `CGRect.null` for the "invisible" state
```suggestion
            resizeHandle.frame = .null
```